src/extension.ts

```diff
@@ -1941,12 +1941,30 @@ function isLintable(document: vscode.TextDocument): boolean {
     return document.languageId === 'yaml' || document.languageId === 'json' || document.languageId === 'helm';
 }
 
+function linterEnabled(enabledLinters: string[], name: string): boolean {
+    for (const linter of enabledLinters) {
```

Comment:
Pretty sure you can use `some` or even `indexOf` here:

`return enabledLinters.some((l) => l === name);`

or

`return enabledLinters.indexOf(name) >= 0;`

Reply:
done.